extensions/ql-vscode/src/model-editor/model-editor-module.ts

```diff
@@ -125,17 +129,24 @@ export class ModelEditorModule extends DisposableObject {
       }
 
       const language = db.language;
-      if (
-        !SUPPORTED_LANGUAGES.includes(language) ||
-        !isQueryLanguage(language)
-      ) {
+
+      // Ruby is only enabled when the config setting is set
+      const isSupportedLanguage =
```

Comment:
If we're making changes anyway, then pulling this little bit out to a function would be good. That way it'll be more reusable if we need to do this in other places.

It could also help to make the `SUPPORTED_LANGUAGES` constant a bit less error-prone. Right now we have a constant which doesn't really do what it sounds like it does because you need to remember to also check Ruby. If we moved it all into a `isSupportedLanguage` function that would be safer.

Reply:
Thanks for the suggestion, that does seem less error-prone.